usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -318,6 +318,46 @@ def award_financial_derivations(derived_fields):
         "award__latest_transaction__contract_data__funding_office_name",
         "award__latest_transaction__assistance_data__funding_office_name",
     )
+    derived_fields["recipient_duns"] = Coalesce(
+        "award__latest_transaction__contract_data__awardee_or_recipient_uniqu",
+        "award__latest_transaction__assistance_data__awardee_or_recipient_uniqu",
+    )
+    derived_fields["recipient_name"] = Coalesce(
+        "award__latest_transaction__contract_data__awardee_or_recipient_legal",
+        "award__latest_transaction__assistance_data__awardee_or_recipient_legal",
+    )
+    derived_fields["recipient_parent_duns"] = Coalesce(
+        "award__latest_transaction__contract_data__ultimate_parent_unique_ide",
+        "award__latest_transaction__assistance_data__ultimate_parent_unique_ide",
+    )
+    derived_fields["recipient_parent_name"] = Coalesce(
+        "award__latest_transaction__contract_data__ultimate_parent_legal_enti",
+        "award__latest_transaction__assistance_data__ultimate_parent_legal_enti",
+    )
+    derived_fields["recipient_country"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_country_code",
+        "award__latest_transaction__assistance_data__legal_entity_country_code",
+    )
+    derived_fields["recipient_state"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_state_code",
+        "award__latest_transaction__assistance_data__legal_entity_state_code",
+    )
+    derived_fields["recipient_county"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_county_name",
+        "award__latest_transaction__assistance_data__legal_entity_county_name",
+    )
+    derived_fields["recipient_city"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_city_name",
+        "award__latest_transaction__assistance_data__legal_entity_city_name",
+    )
+    derived_fields["recipient_congressional_district"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_congressional",
+        "award__latest_transaction__assistance_data__legal_entity_congressional",
+    )
+    derived_fields["recipient_zip_code"] = Coalesce(
+        "award__latest_transaction__contract_data__legal_entity_zip_last4",
+        "award__latest_transaction__assistance_data__legal_entity_zip_last4",
```

Comment:
@BrianZito In case the message was lost: https://github.com/fedspendingtransparency/usaspending-api/pull/2199/files#r364348581